### ISSUE TITLE
Ipad-Ansicht

### ISSUE THREAD

**Issue by McFly753** [Mojo1917/LocBase#178]
Können wir die App auch Ipad-tauglich machen?
Jetzt ist es einfach nur eine vergrößerte Iphone-Ansicht (im falschen Format und verpixelter Schrift)
![img_0015](https://user-images.githubusercontent.com/493612/43986303-5389c0d0-9d0f-11e8-8c63-fb08e51b84c7.PNG)

**Comment by Mojo1917** [comment]
Hier ein Beispiel der neuen Ansichten. Man sieht hier deutlich, dass viele Teil jetzt einfach nur gestreckt werden.
![simulator screen shot - ipad pro 10 5-inch - 2018-08-11 at 10 25 21](https://user-images.githubusercontent.com/20678932/43989782-e01fe222-9d50-11e8-884d-88a438798bb5.png)

**Comment by McFly753** [comment]
Ja, stell es dann mal so hoch, dann kann ich mir Gedanken machen...

**Comment by McFly753** [comment]
schau ich mir noch an

**Comment by McFly753** [comment]
ist ok so, lassen wir erstmal so
